### ISSUE TITLE
Batch Mode... is possible?

### ISSUE THREAD

**Issue by kdantas** [torrent-file-editor/torrent-file-editor#124]
Hi, is there any way to run TFE in batch mode? I need to change the info hash of several .torrent files. If not possible, is there any script to do this? Thanks.

**Comment by drizt** [comment]
Hi. 
In short you can't change info hash that's it.
In long info hash value based on contents of files from whitch consist torrent. So if you need to change hash you must change files. You can add, remove, reorder files.

**Comment by kdantas** [comment]
Hi drizt, are you the creator of the TFE?

**Comment by drizt** [comment]
Yes, it's me. Via command line you can only convert .torrrent file to .json in both directions.
